lib/vm.js

```diff
@@ -426,12 +426,3 @@ module.exports = {
   compileFunction,
   measureMemory,
 };
-
-if (require('internal/options').getOptionValue('--experimental-vm-modules')) {
```

Comment:
maybe leave a comment here or in internal/vm/module.js saying where it gets required from?

Reply:
Done